src/python/pants/help/help_info_extracter.py

```diff
@@ -96,6 +96,7 @@ class OptionHelpInfo:
     removal_version: If deprecated: The version at which this option is to be removed.
     removal_hint: If deprecated: The removal hint message registered for this option.
     choices: If this option has a constrained set of choices, a tuple of the stringified choices.
+    fromfile: Whether the option value should be loaded from a file specified with the `@filepath`.
```

Comment:
I think the file source is optional, indicated with the leading `@`.. (i.e. "could be loaded" rather than "should")

Reply:
Right, I mean if user has provided a string with the `@` then the value will be attempted to be loaded from that file. Perhaps better to say "value will be loaded"?

Reply:
As phrased (also with "value will be loaded") seems to imply it's the only way.
How about: "Supports reading the option value from file when using `@filepath`" ?

Reply:
I trust you this is a better wording, done!